packages/rainbowkit/src/index.ts

```diff
@@ -4,6 +4,7 @@ export { RainbowKitProvider } from './components/RainbowKitProvider/RainbowKitPr
 export { getDefaultConfig } from './config/getDefaultConfig';
 export { getDefaultWallets } from './wallets/getDefaultWallets';
 export { getWalletConnectConnector } from './wallets/getWalletConnectConnector';
+export { getInjectedConnector } from './wallets/getInjectedConnector';
```

Comment:
Shouldn't need to export this

Reply:
There was a discussion on this - https://github.com/rainbow-me/rainbowkit/discussions/1835#discussion-6338498

Reply:
If you check the screenshot shared in the above message - https://github.com/rainbow-me/rainbowkit/pull/1837#issuecomment-1986014696, the dApp was not able to create a wallet connect as this function was not exposed to the user.